core/runtime/jsruntime.rs

```diff
@@ -464,7 +474,11 @@ pub struct CreateRealmOptions {
 impl JsRuntime {
   /// Only constructor, configuration is done through `options`.
   pub fn new(mut options: RuntimeOptions) -> JsRuntime {
-    JsRuntime::init_v8(options.v8_platform.take(), cfg!(test));
+    JsRuntime::init_v8(
+      options.v8_platform.take(),
+      cfg!(test),
+      options.unsafe_expose_natives_and_gc,
```

Comment:
Could you limit it to `cfg!(test)` and unsafe_expose_natives_and_gc - let's not expose a footgun to users

Reply:
#[cfg(test)] doesn't seem to work but I'll investigate options.

Reply:
How about:
```
JsRuntime::init_v8(
      options.v8_platform.take(),
      cfg!(test),
      cfg!(test) && options.unsafe_expose_natives_and_gc,
)
```

Reply:
Sounds good, it's a reasonable solution